brianlong/www.validators.app
# 183408113 - active stake and circulating supply

#### What's this PR do?
PR adds the circulating supply and active stake data in home page.

#### How should this be manually tested?
Make sure to have `daemons/validator_score_mainnet_v1_daemon.rb` running.
Check if the active stake total is consistent with 
Stats::ValidatorHistory.new("mainnet", Batch.last.uuid).total_active_stake.
Circulating Supply data can be pulled only via web3 and this request is being induced for each 5 seconds in the home page.

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183408113)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
